storage_service/locations/models/package.py

```diff
@@ -1010,8 +1011,12 @@ def _create_pointer_file_write_to_disk(
         __, compression_program_version, archive_tool = _get_compression_details_from_premis_events(
             premis_events, self.uuid
         )
-        premis_object = self._create_aip_premis_object(
-            checksum_algorithm, checksum, archive_tool, compression_program_version
+        premis_object = premis.get_aip_premis_object(
+            self,
+            checksum_algorithm,
```

Comment:
Now the package is being passed into the function, are all of these args needed? E.g. will checksum algorithm be available as `package.DEFAULT_CHECKSUM_ALGORITHM`? 

Same question for below's replication package object creation.